config/rabbitmq/rabbitmq.conf

```
@@ -0,0 +1,3 @@
+#more https://www.rabbitmq.com/configure.html#config-file
+
+web_stomp.tcp.port = 15674
```

```suggestion
web_stomp.tcp.port = 15674

```
New line at end of file will make next diffs look better.